packages/remirror__extension-code-block/src/formatter.ts

```diff
@@ -110,37 +112,52 @@ export function formatter(props: FormatterProps): FormattedContent | undefined {
     return { ...result, cursorOffset: result.cursorOffset + increment };
   };
 
+  let parser: BuiltInParserName;
+
+  switch (language) {
+    case 'typescript':
+    case 'ts':
+    case 'tsx':
+      parser = 'typescript';
```

Comment:
Could we pull this out into a separate helper function, then use `return` instead of `break`?

i.e.

```ts
switch (language) {
    case 'typescript':
    case 'ts':
    case 'tsx':
      return 'typescript';
```